software/py/blood_graph.py

```diff
@@ -74,18 +302,19 @@ def get_tg_dim(self, traces):
     
     self.xdim = self.xmax-self.xmin+1
     self.ydim = self.ymax-self.ymin+1
+    return
 
 
-  # private function
+  # private method
   # initialize image
   def init_image(self):
```

Comment:
If something is private, you can add the prefix __ to the name, and make it (effectively) private.

e.g. `def __init_image`

Reply:
Good point, I fixed all private methods with this. Thanks!